doc/api/worker.md

```diff
@@ -0,0 +1,466 @@
+anymore (even if they are not contained in `value`).
+
+If `value` contains [`SharedArrayBuffer`][] instances, those will be accessible
+from either thread.
+
+`value` may still contain `ArrayBuffer` instances that are not in
+`transferList`; in that case, the underlying memory is copied rather than moved.
+
+For more information on the serialization and deserialization mechanisms
+behind this API, see the [serialization API of the `v8` module][v8.serdes].
+
+*Note*: Because the object cloning uses the structured clone algorithm,
+non-enumberable properties, property accessors, and object prototypes are
+not preserved. In particular, [`Buffer`][] objects will be read as
+plain [`Uint8Array`][]s on the receiving side.
+
+### port.ref()
+<!-- YAML
+added: REPLACEME
+-->
+
+Opposite of `unref`, calling `ref` on a previously `unref`d port will *not*
+let the program exit if it's the only active handle left (the default behavior).
+If the port is `ref`d calling `ref` again will have no effect.
+
+If listeners are attached or removed using `.on('message')`, the port will
+be `ref()`ed and `unref()`ed automatically depending on whether
+listeners for the event exist.
+
+### port.unref()
+<!-- YAML
+added: REPLACEME
+-->
+
+Calling `unref` on a port will allow the thread to exit if this is the only
+active handle in the event system. If the port is already `unref`d calling
+`unref` again will have no effect.
+
+If listeners are attached or removed using `.on('message')`, the port will
+be `ref()`ed and `unref()`ed automatically depending on whether
+listeners for the event exist.
+
+### port.start()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {undefined}
+
+Starts receiving messages on this `MessagePort`. When using this port
+as an event emitter, this will be called automatically once `'message'`
+listeners are attached.
+
+## Class: Worker
+<!-- YAML
+added: REPLACEME
+-->
+
+The `Worker` class represents an independent JavaScript execution thread.
+Most Node.js APIs are available inside of it.
+
+Notable differences inside a Worker environment are:
+
+- The [`process.stdin`][], [`process.stdout`][] and [`process.stderr`][]
+  may be redirected by the parent thread.
+- The [`require('worker').isMainThread`][] property is set to `false`.
+- The [`require('worker').postMessage()`][] method is available and the
+  [`require('worker').on('workerMessage')`][] event is being emitted.
+- [`process.exit()`][] does not stop the whole program, just the single thread,
+  and [`process.abort()`][] is not available.
+- [`process.chdir()`][] as well as `process` methods that set group or user ids
+  are not available.
+- [`process.env`][] is a read-only reference to the environment variables.
+- [`process.title`][] can not be modified.
+- Signals will not be delivered through [`process.on('...')`][Signals events].
+- Native addons are not supported.
+- Execution may stop at any point as a result of the [`worker.terminate()`][]
+  method being invoked.
```

Comment:
Nit/suggestion: Remove `the` on previous line and `method` on this line? (It's obviously a method due to the `()`.)